lua/entities/gmod_wire_expression2/core/custom/prop.lua

```diff
@@ -832,6 +832,18 @@ e2function void entity:propShadow(number shadowEnable)
 	this:DrawShadow( shadowEnable ~= 0 )
 end
 
+e2function void entity:propSleep(number sleep)
+	if not ValidAction(self, this, "sleep") then return end
+	local phys = this:GetPhysicsObject()
+	if IsValid(phys) then
```

Comment:
```suggestion
	if phys:IsValid() then
```

`phys` will never be nil.

Reply:
But almost all other code also uses IsValid instead of :IsValid

Reply:
Almost all other code is wrong, because this is an easy minor optimization.